### ISSUE TITLE
[icerec] Incremental Release V05-01-10 (Trac #2101)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2093]
Pass2 needs a fix for the FSS filter since we also apply this filter on IC79 data. The new incremental release needs to include the following commit: http://code.icecube.wisc.edu/projects/icecube/changeset/158703/IceCube

The modifications on the FSS filter have been discussed with Hrvoje, Erik, and on the last BSM call.

Migrated from https://code.icecube.wisc.edu/ticket/2101
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:14:49", 
    "description": "Pass2 needs a fix for the FSS filter since we also apply this filter on IC79 data. The new incremental release needs to include the following commit: http://code.icecube.wisc.edu/projects/icecube/changeset/158703/IceCube\n\nThe modifications on the FSS filter have been discussed with Hrvoje, Erik, and on the last BSM call.", 
    "reporter": "joertlin", 
    "cc": "hdujmovic", 
    "resolution": "fixed", 
    "_ts": "1550067289977126", 
    "component": "combo reconstruction", 
    "summary": "[icerec] Incremental Release V05-01-10", 
    "priority": "blocker", 
    "keywords": "pass2, FSSFilter, IC79", 
    "time": "2017-10-17T15:47:02", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20171019T17:03:32: 
* olivas commented:

> In changeset 158912:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="158912"
> updated externals and release notes.  closes #2093.
> ```


* olivas changed resolution from "" to "fixed"
* olivas changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:14:49: 
* olivas commented:

> Milestone October 2017 deleted


* olivas changed milestone from "October 2017" to ""
